src/cli/cluster/cluster_manager.js

```diff
@@ -86,7 +86,6 @@ module.exports = class ClusterManager {
     const fromRoot = require('../../utils/fromRoot');
 
     this.watcher = chokidar.watch([
-      'src/plugins',
```

Comment:
it might make sense to upgrade chokidar too, they are at 1.4.3 and may include fixes for related issues.